docs/userguide/strings/properties.md

```diff
@@ -82,100 +90,100 @@ words, there are two ICU properties for the same Unicode property, one
 delivering single values (for per-code point lookup) and the other delivering
 sets of values (for use with value aliases and UnicodeSet).
 
-| UCD Name(see PropertyAliases.txt) | Type |  | ICU4C uchar.hICU4J UCharacter | UCD File (.txt) |
```

Comment:
Why did you remove the fifth column "UCD File (.txt)"?

Reply:
I see the comment in the email thread:

> * strings/properties.md. The big properties table is too wide, and the last column is not visible. I intend to change this to a series of short sections.

Reply:
I worked on the table to make it less wide, and ended up with a half-visible last column. Then I decided that it was both tedious and unnecessary to document *on this page* which property has data in which Unicode file. Instead, I add a short paragraph pointing to UAX 44 which has all that and is carefully kept in sync with the properties for each Unicode release.